### DISCUSSION TITLE
how can I override <leader>/ to bind comment out

### DISCUSSION THREAD

**Discussion by soomtong** [LazyVim/LazyVim#1731] [Q&A]
I try many times, and find questions and answers. :0

in my `config/keymaps.lua` like this.

```lua
vim.keymap.set("n", "<leader>/", "gcc", { remap = true, silent = true, desc = "Comment line" })
vim.keymap.set("x", "<leader>/", "gc", { remap = true, silent = true, desc = "Comment selection" })
```

but it shows default grep.

![image](https://github.com/LazyVim/LazyVim/assets/906974/89dbe78c-a079-450d-9be5-bc3e609a2b90)

but it's ok in selection mode :) 

![image](https://github.com/LazyVim/LazyVim/assets/906974/d615f1f7-5c74-4af4-a204-e76fd0585cc4)

can I fix or make better?

**Comment by soomtong** [comment]
oops. it's working.

I remove all `.local` LazyVim files. yea, like clean install.

![image](https://github.com/LazyVim/LazyVim/assets/906974/f8ef8db7-e329-4d45-8a5c-c94c66c36afd)

shame, but someone get helps by my fault.

**Comment by soomtong** [comment]
I found it cuz of, lazy loading of Telescope.

just load eager it. may solved.

```lua
  { "nvim-telescope/telescope.nvim", lazy = false },
```

**Reply by rjshrjndrn** [reply]
I did the same. But is this correct @dpetka2001

**Reply by dpetka2001** [reply]
What do you want to do?

**Reply by rjshrjndrn** [reply]
Ha.. Got the solution. Stupid me.

https://www.lazyvim.org/configuration/plugins#%EF%B8%8F-adding--disabling-plugin-keymaps

In this specific case,

```lua
-- lua/plugins/telescope.lua
return {
  "nvim-telescope/telescope.nvim",
  keys = {
    -- disable the keymap to grep files
    {"<leader>/", false},
    },
  }
  ```
